src/hotspot/share/gc/g1/g1CollectedHeap.cpp

```diff
@@ -1406,8 +1406,8 @@ G1CollectedHeap::G1CollectedHeap() :
   _summary_bytes_used(0),
   _bytes_used_during_gc(0),
   _archive_allocator(NULL),
-  _survivor_evac_stats("Young", YoungPLABSize, PLABWeight),
-  _old_evac_stats("Old", OldPLABSize, PLABWeight),
+  _survivor_evac_stats("Young", YoungPLABSize, YoungPLABSize * ParallelGCThreads, PLABWeight),
+  _old_evac_stats("Old", OldPLABSize, OldPLABSize * ParallelGCThreads, PLABWeight),
```

Comment:
I think you could leave those two unchanged and in the constructor for `G1EvacStats` do the multiplication with `ParallelGCThreads` for the desired value.

Reply:
And rename the `default_plab_sz` parameter to `default_per_thread_plab_size` (and remove the `desired_net_plab_sz_` as suggested).